runtime.md

```diff
@@ -121,11 +121,18 @@ When the process in the container is stopped, irrespective of it being as a resu
 This operation MUST generate an error if it is not provided the container ID.
 Attempting to delete a container that does not exist MUST generate an error.
 Attempting to delete a container whose process is still running MUST generate an error.
-Deleting a container MUST delete the resources that were created during the `create` step.
-Note that resources associated with the container, but not created by this container, MUST NOT be deleted.
 Once a container is deleted its ID MAY be used by a subsequent container.
 
+Deleting a container MUST delete the resources that were created during the `create` step.
+Resources associated with the container, but not created by this container, MUST NOT be deleted.
+
+On Linux, [`mounts`](config.md#mounts), [`root`](config.md#root), [`linux.devices`](config-linux.md#devices), and other filesystem changes belong to the container which created the [container mount namespace][container-namespace3].
+If a container joins an existing mount namespace and applies those settings, deleting the container MUST NOT reverted the settings.
+Similarly [`hostname`](config.md#hostname) changes belong to the container which created the container UTS namespace.
+[`linux.resources`](config-linux.md#control-groups) settings belong to the container which created the associated cgroup.
```

Comment:
> WDYT about something like…

I'm fine with a bit more hand-holding like that.  To be absolutely clear that these examples were informative (and not adding new normative restrictions), I've reworded them to avoid MUST and split them off into new subsections with 083f23e → 52a5199.  I've also added a similar cgroup example so we don't focus exclusively on namespaces.  And I've also filled out the configs because I don't like fragments (although we have an existing fragment example [here][1]).  I personally prefer the compactness of 083f23e, but maybe maintainers prefer 52a5199 or a sweet spot somewhere in between?  If you want something in the middle, please give me a list of corners to cut (e.g. “drop the subsections” or “use JSON fragments for the configs”).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/config-linux.md#example-3